src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -808,8 +807,6 @@ internal void SetPackageState(PackageDownloadHandle packageDownloadHandle, strin
             {
                 PackageManagerExtension.PackageLoader.LoadPackages(new List<Package> { dynPkg });
                 packageDownloadHandle.DownloadState = PackageDownloadHandle.State.Installed;
-
-                dynPkg.LoadState.SetAsLoaded();
```

Comment:
This was wrong.
LoadState is set inside the PackageManagerExtension.PackageLoader.LoadPackages method (which can be different from Loaded)

Reply:
So it's possible for package download handle state to say "installed" but the package `loadstate` to be unloaded or error?

Reply:
Yes, it can. 
The PackageDownloadHandle's downloadState does not handle "load into library" failures that well
We can definitely make it keep count of  "loadState" but that may be something that needs some UX thought ...